src/js/utils.js

```diff
@@ -68,7 +68,7 @@ export function memoize(_target, _property, descriptor) {
  */
 export const html = (strings, ...args) =>
     strings
-        .map((str, i) => [str, args[i]])
+        .map((str, i) => [str.trim(), args[i]?.trim()])
```

Comment:
I noticed that in at least some places, setting `innerText` to a

```
html`<whatever>...</something>`
```

string was creating a list of n element children alternating with n+1 empty text nodes padded in between. This seems like a potential performance issue (more nodes === bad, right?), so I `trim` them out here. I didn't see a visual difference on making this change, but was the old behavior intended?

Reply:
@texodus I reverted these changes, since they were breaking some of the perspective tests (the white space in some of the expected results changed slightly).

I would still like to know:
- What's the role of the filler whitespace text nodes?
- Is there a possible performance concern here, or are the surplus nodes a non-issue?